docs-site/docs/user/project/projects/bot-accounts/manage-bot-personal-access-tokens.md

```diff
@@ -4,8 +4,47 @@ id: manage-bot-PAT
 title: Manage Bot Account Personal Access Tokens
 ---
 
-<!-- TODO: Add headers with description/explanations of following:
-- View PAT
-- Generate PAT
-- Revoke PAT
--->
+Personal Access Tokens (PAT) are needed to use the IRIDA Next GraphQL API. See [Authentication with Personal Access Tokens](/docs/extend/graphql#authentication-with-personal-access-tokens) for more information.
+
+## View PAT
+
+Prerequisite:
+
+- You must have at least a **Maintainer** role.
+
+To view a bot account PAT within a project:
```

Comment:
Instead of PAT please use whole words here and anywhere below. The PAT here would be `personal access tokens` since the bot account could have multiple tokens. This will also apply to the groups -> manage bot personal access tokens docs

Reply:
The acronym is defined at is at the top of the page, `Personal Access Tokens (PAT)`. I can get this in the next commit.